### PR TITLE
Drop Node 6, 8, 9, 11, and 13 support.

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "rsvp": "^4.8.3"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || 10.* || >= 12.*"
```
Comment (stefanpenner) [inline package.json]: This also drops support for node 11, which doesn't EOL until 2019-06-01. Was that the intention?

Comment (rwjblue) [reply package.json]: Yeah, but I can roll it back

Comment (rwjblue) [reply package.json]: Updated to bring back Node 11 support

Comment (SergeAstapov) [reply package.json]: 2019-06-01 has passed, shall Node.js 11 support be dropped?

File: appveyor.yml [hunk 2]
```diff
@@ -4,8 +4,8 @@
 environment:
   MOCHA_REPORTER: "mocha-appveyor-reporter"
   matrix:
-    - nodejs_version: "6"
-    - nodejs_version: "8"
+    - nodejs_version: "10"
```
Comment (scalvert) [inline appveyor.yml]: Shouldn't this include node 8, as per the .travis.yml?

**Pull request merged**